templates/compositions/ec-component-icons-with-text/icons-with-text.html.twig

```diff
@@ -0,0 +1,46 @@
+{% spaceless %}
+
+{#
+  /**
+   * @file
+   * Theme implementation for a ECL list of icons with text.
+   */
+Parameters:
+- "items" (array) (default: [])
+  - "icon" (string) (default: '')
+  - "text" (string) (default: '')
+#}
+
+{# Internal properties #}
+
+{% set _icon_path = icon_path|default('') %}
+{% set _extra_classes = 'ecl-u-mr-s' %}
+
+{# Internal logic - Process properties #}
+
+{% set _items = [] %}
+{% for item in  items %}
+  {% if item.icon and item.text %}
+    {% set _items = _items|merge([item]) %}
+  {% endif %}
+{% endfor %}
+
+{# Print the result #}
+
+{% if _items %}
+<ul class="ecl-unordered-list ecl-unordered-list--no-bullet">
+  {% for _item in _items %}
+    <li class="ecl-u-d-flex ecl-u-align-items-center ecl-unordered-list__item{% if not loop.first %} ecl-u-mt-l{% endif %}">
+      {% include '@ecl/icon' with {
+        icon: {
+          path: _icon_path,
+          type: 'general',
+          name: _item.icon
```

Comment:
We should fallback to `document` if none specified.

Reply:
Done. Used `file` icon